### PR TITLE
GO SDK for Router Firewall Groups

### PR DESCRIPTION
**Pull request by manjunath-batakurki**

### CODE REVIEW

**Review by gandharvas — DISMISSED** [review]

File: pkg/client/router.go [hunk 1]
```diff
@@ -209,3 +209,63 @@ func (r *RouterAPIService) DeleteRouterNat(
...
 	return natResp, err
 }
+
+func (r *RouterAPIService) CreateRouterFirewallRuleGroup(
+	ctx context.Context,
+	routerID int,
+	request models.CreateRouterFirewallRuleGroupRequest,
+) (models.CreateRouterFirewallRuleGroupResponse, error) {
+	firewallGroupResp := models.CreateRouterFirewallRuleGroupResponse{}
+	serverAPI := &api{
```
Comment (tshihad) [inline pkg/client/router.go:219]: routerAPI := &api{

File: pkg/client/router.go [hunk 2]
```diff
@@ -209,3 +209,63 @@ func (r *RouterAPIService) DeleteRouterNat(
...
+
+	return firewallGroupResp, err
+}
+
+func (r *RouterAPIService) GetSpecificRouterFirewallRuleGroup(
+	ctx context.Context,
+	routerID, firewallGroupID int,
+) (models.GetSpecificRouterFirewallRuleGroupResponse, error) {
+	firewallGroupResp := models.GetSpecificRouterFirewallRuleGroupResponse{}
+	serverAPI := &api{
```
Comment (tshihad) [inline pkg/client/router.go:238]: +1, can you please update all serverAPI to routerAPI in this file as well

**Review by tshihad — COMMENTED** [review]
Please fix typo

**Review by tshihad — DISMISSED** [review]
LGTM

**Comment by manjunath-batakurki** [comment]
Branch is not proper, Hence closing it

**Pull request closed without merging**